minio/minio
# Add config to store subnet license

## Description

Command to set subnet license:

`mc admin config set {alias} subnet license={token}`

Signed-off-by: Shireesh Anjal <shireesh@minio.io>

## Motivation and Context

Will be helpful in implementing `mc admin subnet register` where mc will fetch the cluster's license and then store in minio config for future use.

## How to test this PR?

- Run `mc admin config set/get` commands against minio built with the PR e.g.
 - to set the license:
`mc admin config set {alias} subnet license={token}`
 - Then get the value and verify that it is same as the one set above
`mc admin config get {alias} subnet`

## Types of changes

## Checklist:
